framework/helpers/BaseHtml.php

```diff
@@ -660,12 +660,21 @@ public static function fileInput($name, $value = null, $options = [])
      * the attributes of the resulting tag. The values will be HTML-encoded using [[encode()]].
      * If a value is null, the corresponding attribute will not be rendered.
      * See [[renderTagAttributes()]] for details on how attributes are being rendered.
+     * The following special options are recognized:
+     *
+     * - `doubleEncode`: whether to encode HTML entities in `$value`. If false, HTML entities in `$value` will not
+     * be further encoded.
+     *
      * @return string the generated text area tag
      */
     public static function textarea($name, $value = '', $options = [])
     {
         $options['name'] = $name;
-        return static::tag('textarea', static::encode($value), $options);
+        $doubleEncode = (isset($options['doubleEncode'])) ? $options['doubleEncode'] : true;
```

Comment:
Ahh nice, haven't used `ArrayHelper` enough to know you could do that. I have made the changes locally.